pkg/runtime/scheme.go

```diff
@@ -153,6 +153,38 @@ func (s *Scheme) AddUnversionedTypes(version unversioned.GroupVersion, types ...
 	}
 }
 
+func validateListType(t reflect.Type) {
```

Comment:
This method should be in helpers with the other helpers that do list validation.